app/src/org/commcare/tasks/DataPullTask.java

```diff
@@ -441,13 +436,6 @@ private void storeSuccessfulSyncTime(SharedPreferences prefs) {
         } catch (StorageFullException e) {
             e.printStackTrace();
             failureReason = e.getMessage();
-        }
-
-        //These last two aren't a sign that the incoming data is bad, but
-        //we still can't recover from them usefully
-        catch (SessionUnavailableException e) {
-            e.printStackTrace();

```

Comment:
Good call, there is chance it occurs when trying to pull data on login. I'll add it back